### PR TITLE
Add: Building 21.4 from Source on Fedora

### PR DESCRIPTION
**Pull request by pascalholthaus**
**What**:
Add building 21.4 from Source on Fedora

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: src/21.4/source-build/gsa/dependencies.rst [hunk 1]
```diff
@@ -19,4 +19,12 @@
...
-.. _yarn: https://classic.yarnpkg.com/
+    .. _yarn: https://classic.yarnpkg.com/
+
+  .. tab:: Fedora
+    .. code-block::
+      :caption: Install nodejs 14
+
+      sudo dnf module enable nodejs:14 -y
+      sudo dnf install -y nodejs yarnpkg nodejs-typescript
+
```
Comment (bjoernricks) [inline src/21.4/source-build/gsa/dependencies.rst:30]: Needs to be below the content I think. It is a reference for the link URL use above.
```suggestion

  .. tab:: Fedora
    .. code-block::
      :caption: Install nodejs 14

      sudo dnf module enable nodejs:14 -y
      sudo dnf install -y nodejs yarnpkg nodejs-typescript

.. _yarn: https://classic.yarnpkg.com/
```

File: src/21.4/source-build/gvm-libs/dependencies.rst [hunk 2]
```diff
@@ -1,21 +1,46 @@
...
+        gpgme-devel \
+        gnutls-devel \
+        libgcrypt-devel \
+        libuuid-devel \
+        libssh-devel \
+        hiredis-devel \
+        libxml2-devel \
+        libpcap-devel \
+        libnet-devel \
+        paho-c-devel
```
Comment (bjoernricks) [inline src/21.4/source-build/gvm-libs/dependencies.rst:39]: We don't have mqtt in 21.4 thus paho is not necessary.
```suggestion
        libnet-devel
```

File: src/21.4/source-build/ospd-openvas/dependencies.rst [hunk 3]
```diff
@@ -1,15 +1,37 @@
...
+        python3-setuptools \
+        python3-packaging \
+        python3-wrapt \
+        python3-cffi \
+        python3-psutil \
+        python3-lxml \
+        python3-defusedxml \
+        python3-paramiko \
+        python3-redis \
+        python3-paho-mqtt
```
Comment (bjoernricks) [inline src/21.4/source-build/ospd-openvas/dependencies.rst:36]: Same here. No mqtt in 21.4
```suggestion
        python3-redis
```

File: src/21.4/source-build/gsa/dependencies.rst (hunk 1 above)
Comment (pascalholthaus) [reply src/21.4/source-build/gsa/dependencies.rst]: done

File: src/21.4/source-build/gvm-libs/dependencies.rst (hunk 2 above)
Comment (pascalholthaus) [reply src/21.4/source-build/gvm-libs/dependencies.rst]: done

File: src/21.4/source-build/ospd-openvas/dependencies.rst (hunk 3 above)
Comment (pascalholthaus) [reply src/21.4/source-build/ospd-openvas/dependencies.rst]: done

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**